### ISSUE TITLE
Automatically create Changelog

### ISSUE THREAD

**Issue by metmajer** [opendevstack/ods-project-quickstarters#397]
**Is your feature request related to a problem? Please describe.**
Maintaining Changelogs at release time is a time-consuming task. This could be automated.

**Describe the solution you'd like**
At [terraform-docs](https://github.com/segmentio/terraform-docs), as many other open source projects, we use the [git-chglog](https://github.com/git-chglog/git-chglog) generator to do just that. A sample config and template can be found [here](https://github.com/segmentio/terraform-docs/tree/master/.chglog).

FYI @gerardcl

**Comment by gerardcl** [comment]
@segator was also proposing a tool more integrated to github, let's check that too!

**Comment by michaelsauter** [comment]
@metmajer Yes! That would be so awesome. Can you explain a little how your process looks like so that git-chglog can generate the changelog well? E.g. how do you deal with labels/milestones/projects? And how do you deal with issues vs. PRs?
